.travis.yml

```diff
@@ -2,14 +2,6 @@
 sudo: required
 
 install:
-  # Docker 1.13 adds "--cache-from" which really speeds up our Travis builds.
-  # Travis is currently several versions behind: When/if they catch up,
-  # this can all go away, and we can stop sudoing docker.
-  - sudo apt-get install libapparmor1
-  - DEB=docker-engine_1.13.0-0~ubuntu-precise_amd64.deb
-  - wget https://apt.dockerproject.org/repo/pool/main/d/docker-engine/$DEB
-  - sudo dpkg -i $DEB
-  - sudo docker --version
```

Comment:
I guess Travis is now providing Docker? I'd like to keep the version in there, to help with debuggging, and to fail fast if it's not installed.